packages/assets-controllers/src/AccountTrackerController.test.ts

```diff
@@ -42,6 +49,8 @@ describe('AccountTrackerController', () => {
     const controller = new AccountTrackerController({
       onPreferencesStateChange: sinon.stub(),
       getIdentities: () => ({}),
+      getSelectedAddress: () => '0x0',
```

Comment:
Q: why using 0x0 address for the test? This is a very special one that will never be used in a wallet and that may trigger some specific behaviour. Should we use a random valid address instead?